DeductoActivity.py

```diff
@@ -87,8 +72,13 @@ def __init__(self, handle):
         self._game = Game(canvas, parent=self, colors=self.colors)
 
         self._sharing = False
-        self._initiating = False
-        self._setup_presence_service()
+        self._initiating = None
```

Comment:
May I know why this was changed?

Reply:
Thanks for noticing , I find out that it was a mistake. I will change this one.